usaspending_api/awards/models_matviews.py

```diff
@@ -355,7 +355,7 @@ class Meta:
 
 
 class AwardMatview(models.Model):
-    generated_unique_award_id = models.TextField()
+    generated_unique_award_id = models.TextField(primary_key=True)
```

Comment:
Was this for testing JOINS in the future?

Reply:
This was a solution to querying the matview with Django's ORM. Without it, doing `AwardMatview.objects.filter(...)` would error out as the ORM automatically includes and ID column if no `primary_key=True` is provided.

Reply:
Good to know. That might have been useful for me two weeks ago